client/vite.config.js

```diff
@@ -8,9 +8,13 @@ export default defineConfig({
     globals: true,
     setupFiles: './tests/setup.js',
     environment: 'jsdom',
+
+    include: ['src/tests/**/*.{test,spec}.{js,jsx,ts,tsx}'],
```

Comment:
We also do not need this line, kindly remove it. The `vite.config.js` file is adequate for now.